tests/foreign_object/tests.py

```diff
@@ -391,3 +391,10 @@ def test_batch_create_foreign_object(self):
         """ See: https://code.djangoproject.com/ticket/21566 """
         objs = [Person(name="abcd_%s" % i, person_country=self.usa) for i in range(0, 5)]
         Person.objects.bulk_create(objs, 10)
+
+    def test_error_unsaved_object(self):
```

Comment:
I hadn't seen that this was handled in `many_to_one/tests.py`. Just removed that extra test.